### DISCUSSION TITLE
when use presetWind,how to close bg-opacity?

### DISCUSSION THREAD

**Discussion by ismewumei** [unocss/unocss#2943] [Q&A]
When I used presetWind, use class name "bg-blue",always new --un-bg-opacity:1,like this:

`.bg-blue-300{--un-bg-opacity:1;background-color:rgba(147,197,253,var(--un-bg-opacity));}`

how can I close "bg-opcity".

**Comment by zyyv** [comment]
Try to use `bg-blue/30`.
